local-cluster/src/local_cluster.rs

```diff
@@ -129,9 +131,20 @@ impl LocalCluster {
 
     pub fn new(config: &ClusterConfig) -> Self {
         assert_eq!(config.validator_configs.len(), config.node_stakes.len());
-        let leader_keypair = Arc::new(Keypair::new());
+        let validator_keys = {
+            if let Some(ref keys) = config.validator_keys {
```

Comment:
As far as I know, you should never need `ref` annotations anymore.

Reply:
hmmm still complains of a move of a shared reference unless I annotate it here

Reply:
weird. thanks for checking